test/unit/services/BatchFileUploadUpscanNotificationBusinessServiceSpec.scala

```diff
@@ -75,67 +76,69 @@ class BatchFileUploadUpscanNotificationBusinessServiceSpec extends UnitSpec with
 
       actual shouldBe (())
       verify(mockRepo).update(
+        ameq[UUID](ApiSubscriptionFieldsTestData.subscriptionFieldsId.value).asInstanceOf[SubscriptionFieldsId],
         ameq[UUID](FileReferenceOne.value).asInstanceOf[FileReference],
         ameq(callbackFields))(any[HasConversationId]
       )
       verify(mockConnector).send(ameq(fileTransmissionRequest))
     }
 
     "return failed future when no metadata record found for file reference" in new SetUp {
-      when(mockRepo.update(FileReferenceOne, callbackFields)).thenReturn(Future.successful(None))
+      when(mockRepo.update(ApiSubscriptionFieldsTestData.subscriptionFieldsId, FileReferenceOne, callbackFields)).thenReturn(Future.successful(None))
 
-      val error = intercept[IllegalStateException](await(service.persistAndCallFileTransmission(readyCallbackBody)))
+      val error = intercept[IllegalStateException](await(service.persistAndCallFileTransmission(ApiSubscriptionFieldsTestData.subscriptionFieldsId, readyCallbackBody)))
 
       error.getMessage shouldBe s"database error - can't find record with file reference ${FileReferenceOne.value.toString}"
       verify(mockRepo).update(
+        ameq[UUID](ApiSubscriptionFieldsTestData.subscriptionFieldsId.value).asInstanceOf[SubscriptionFieldsId],
         ameq[UUID](FileReferenceOne.value).asInstanceOf[FileReference],
         ameq(callbackFields))(any[HasConversationId]
       )
       verifyZeroInteractions(mockConnector)
     }
 
     "return failed future when file reference not found in returned metadata" in new SetUp {
-      when(mockRepo.update(FileReferenceOne, callbackFields)).thenReturn(Future.successful(Some(BatchFileMetadataWithFileTwo)))
+      when(mockRepo.update(ApiSubscriptionFieldsTestData.subscriptionFieldsId, FileReferenceOne, callbackFields)).thenReturn(Future.successful(Some(BatchFileMetadataWithFileTwo)))
 
-      val error = intercept[IllegalStateException](await(service.persistAndCallFileTransmission(readyCallbackBody)))
+      val error = intercept[IllegalStateException](await(service.persistAndCallFileTransmission(ApiSubscriptionFieldsTestData.subscriptionFieldsId, readyCallbackBody)))
 
       error.getMessage shouldBe s"database error - can't find file with file reference ${FileReferenceOne.value.toString}"
       verify(mockRepo).update(
+        ameq[UUID](ApiSubscriptionFieldsTestData.subscriptionFieldsId.value).asInstanceOf[SubscriptionFieldsId],
         ameq[UUID](FileReferenceOne.value).asInstanceOf[FileReference],
         ameq(callbackFields))(any[HasConversationId]
       )
       verifyZeroInteractions(mockConnector)
     }
 
     "propagate exception encountered in repo" in new SetUp {
-      when(mockRepo.update(FileReferenceOne, callbackFields)).thenReturn(Future.failed(emulatedServiceFailure))
+      when(mockRepo.update(ApiSubscriptionFieldsTestData.subscriptionFieldsId, FileReferenceOne, callbackFields)).thenReturn(Future.failed(emulatedServiceFailure))
```

Comment:
maybe import `ApiSubscriptionFieldsTestData.subscriptionFieldsId` ? It would look cleaner